packages/ui/src/views/repo/repo.types.ts

```diff
@@ -0,0 +1,25 @@
+export interface BranchSelectorListProps {
+  name: string
+  default?: boolean
+}
+
+export interface BranchProps {
```

Comment:
Let's avoid using the `Props` suffix unless it's being used to describe the props to a component.
```suggestion
export interface BranchSelectorListItem {
  name: string
  default?: boolean
}

export interface Branch {
```